addons/website/tests/test_ui.py

```diff
@@ -216,3 +216,9 @@ def test_11_website_snippet_background_edition(self):
             'mimetype': 'image/png',
         })
         self.start_tour('/', 'snippet_background_edition', login='admin')
+
+    def test_12_edit_translated_page_redirect(self):
+        lang = self.env['res.lang']._activate_lang('nl_NL')
+        self.env['website'].browse(1).write({'language_ids': [(4, lang.id, 0)]})
+        self.start_tour("/nl", 'edit_translated_page_redirect', login='admin')
+        self.start_tour("/nl/contactus", 'edit_translated_page_redirect', login='admin')
```

Comment:
Do we need this second tour?

Reply:
I added the second tour because the homepage is an edge case... So the second tour tests the general case, however the general case wasn't buggy before.

Reply:
Ok it is fine then let's keep both, let's just rename `nl_NL` to `nl` in your js tour

Reply:
I'll add an extra comment in the the test for context